torch/csrc/autograd/profiler.h

```diff
@@ -248,6 +261,10 @@ struct TORCH_API Event final {
     return device_;
   }
 
+  void setDevice(int device) {
```

Comment:
what is the meaning of device? why is it different from c10::Device. If it's just a gpu index - call it `device_index`. In any case - add a comment about semantics

Reply:
Yes it's index. I think maybe device_id will be better than device_index, as it can potentially mean other things.